.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,58 +1,38 @@
-Hello team,
+|Related issue|
+|---|
+||
 
-`<YOUR_INTRODUCTION_TEXT>`
-
-Closes #`<YOUR_ISSUE_NUMBER>`
-
-# Tests logic
+<!--
+This template reflects sections that must be included in new Pull requests.
+Contributions from the community are really appreciated. If this is the case, please add the
+"contribution" to properly track the Pull Request.
 
-`<YOUR_LOGIC_TEST_DESCRIPTION>`
+Please fill the table above. Feel free to extend it at your convenience.
+-->
 
-# Tests checks
+## Description
 
-- [ ] Proven that tests **pass** when they have to pass
-- [ ] Proven that tests **fail** when they have to fail
-- [ ] Proven that tests have the expected behavior in **RPM and DEB**
-- [ ] Checked that **all vulnerability detector tests work correctly** (my changes don't break anything)
-- [ ] Tested and passed in Jenkins. Build URL: `<YOUR_JENKINS_BUILD_URL>`
 <!--
-Important: Don't remove this check if your PR modifies Python code.
+Add a clear description of how the problem has been solved.
 -->
-- [ ] Python codebase satisfies PEP-8 style style guide. `pycodestyle --max-line-length=120 --show-source --show-pep8 file.py`
-
-## RPM manager
-
-- When the expected behavior occurs:
-
-```
-<PYTEST_RESULT>
-```
-
-- When the module fails because of `<YOUR_CAUSE>`:
-
-```
-<PYTEST_RESULT>
-```
 
-## DEB manager
+## Configuration options
 
-- When the expected behavior occurs:
-
-```
-<PYTEST_RESULT>
-```
-
-- When the module fails because of `<YOUR_CAUSE>`:
-
-```
-<PYTEST_RESULT>
-```
+<!--
+When proceed, this section should include new configuration parameters.
+-->
 
-## Integrity check with other tests
+## Logs example
 
+<!--
+Paste here related logs and alerts
+-->
 
-```
-<PYTEST_RESULT>
-```
+## Tests
```

Comment:
We could add a step to add a screenshot of the execution of the test (?)

Reply:
I think the _Logs example_ section covers that